test_mystery_word.py

```diff
@@ -0,0 +1,61 @@
+import mystery_word as myt
+import evil_hangman as evil
+
+
+def test_validate_level():
+    assert (myt.validate_level("E") is True or
```

Comment:
You should use `and` instead of `or` here. I'd probably split this into three asserts.